tests/ui/const-generics/late-bound-vars/in_closure.rs

```diff
@@ -1,4 +1,4 @@
-// failure-status: 101
+// failure-status: 1
```

Comment:
Why did this change?

Reply:
I think this is because this PR unwinds the last panic instead of the first one. The last panic being a `FatalErrorMarker` so we catch it and gracefully exit with 1.